### PR TITLE
Add "Implement Payment Tenders" doc

### PR DESCRIPTION
**Pull request by chriscressman**
* Remove docs being replaced
* Update navigation and cross references
* Keep and update "Test a Credit Card Transaction" doc

WORKAREA-13

### CODE REVIEW

**Comment by chriscressman** [comment]
This is a pretty substantial doc, so I decided to get a draft PR started for it. Over the next day or two, I'll complete the 8 subsections (which I have in various states of completeness right now).

File: docs/source/articles/integrate-payment-tenders.html.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+title: Integrate Payment Tenders
+excerpt: TODO
+created_at: 2019/10/10
+---
+
+Integrate Payment Tenders
+======================================================================
+
+Workarea provides two default tender types: store credit and credit card, although the credit card implementation requires customization to communicate with a credit card payment service provider in production.
+Workarea provides plugins which fully implement the credit card tender type, as well as plugins that extend the platform with additional tender types (e.g. gift card, PayPal).
```
Comment (bencrouse) [inline docs/source/articles/integrate-payment-tenders.html.md]: Might as well link to those plugins.

File: docs/source/articles/integrate-payment-tenders.html.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+( Browse [Workarea plugins tagged "Payment"](https://plugins.workarea.com/plugins?tags=Payment) to see all such plugins. )
```
Comment (bencrouse) [inline docs/source/articles/integrate-payment-tenders.html.md]: Can you omit the plugins.workarea.com link? Not all of those are up-to-date and/or open-source. We need to do a bunch of work on that site.

File: docs/source/articles/integrate-payment-tenders.html.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+* [Set up Gateways for the Tender Type](#set-up-gateways-for-the-tender-type_3)
+* [Implement Card Tokenization](#implement-card-tokenization_4)
+* [Implement Each Payment Operation](#implement-each-payment-operation_5)
+* [Integrate the Type into Workarea Storefront](#integrate-the-type-into-workarea-storefront_6)
+* [Test Your Integration](#test-your-integration_7)
+* [Configure Commerce Cloud](#configure-commerce-cloud_8)
+
+Because tender implementations are diverse, these steps are fairly abstract.
+This doc therefore provides concrete examples to back up each step.
+The examples use the following tender types:
```
Comment (bencrouse) [inline docs/source/articles/integrate-payment-tenders.html.md]: Nice, these are good examples.

File: docs/source/articles/integrate-payment-tenders.html.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+* _Store credit_ and _credit card_ (partial implementation) from [Workarea](https://www.workarea.com/) v3.5.0 (
+  [gem](https://rubygems.org/gems/workarea/versions/3.5.0),
+  [docs](https://developer.workarea.com/),
+  [source](https://github.com/workarea-commerce/workarea/tree/v3.5.0)
+  )
+* _Credit card_ (extensions to base) from [Workarea Braintree](https://plugins.workarea.com/plugins/braintree) v1.0.3 (
```
Comment (bencrouse) [inline docs/source/articles/integrate-payment-tenders.html.md]: Can you do plugin links to Github for now? See notes about home base above.

File: docs/source/articles/integrate-payment-tenders.html.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
+---
+title: Integrate Payment Tenders
+excerpt: TODO
+created_at: 2019/10/10
+---
+
+Integrate Payment Tenders
+======================================================================
+
+Workarea provides two default tender types: store credit and credit card, although the credit card implementation requires customization to communicate with a credit card payment service provider in production.
```
Comment (jyucis) [inline docs/source/articles/integrate-payment-tenders.html.md]: Change: although the credit card implementation requires customization to communicate with a credit card payment service provider in production

to 

although the credit card implementation requires customization to communicate with a payment service provider in production

The "credit card" is implied in first part of the sentence.

**Comment by chriscressman** [comment]
This doc was becoming too abstract and branching too much and was too complicated. I gutted it so I could get a better view of what is going on.

Instead of proceeding with this doc, I'd like to deliver 3 separate docs:

* One focused on "completing" the credit card tender type, using Workarea Braintree as a concrete example
* Another focused on implementing an alternative primary tender type, using Workarea PayPal as a concrete example
* And a third focused on implementing an advance payment tender type, using Workarea Gift Cards as a concrete example

With this solution, the docs won't need any internal branching and they will each be much more concrete and easier to follow.

The downside is it will take a little longer to finish.

Let's discuss at our standup.

**Comment by chriscressman** [comment]
We discussed this, and I'm going to proceed with the 3-doc solution I described above.

**Pull request closed without merging**